.github/workflows/ci.yml

```diff
@@ -19,22 +20,30 @@ jobs:
 
   lint:
     name: Lint
-    runs-on: windows-2019
+    runs-on: windows-2022
```

Comment:
I would suggest to go all the way to `windows-2025` if you are going to update it.